lib/cmock_generator_plugin_ignore.rb

```diff
@@ -61,6 +66,16 @@ def mock_interfaces(function)
     end
     lines << "  Mock.#{function[:name]}_IgnoreBool = (char)1;\n"
     lines << "}\n\n"
+  
+    #add stop ignore function. it does not matter if there are any args
+    lines << "void #{function[:name]}_CMockStopIgnore(void)\n{\n"
+    lines << "  if(Mock.#{function[:name]}_IgnoreBool) {\n"
```

Comment:
Nitpicks:
- Looks like the opening curly brace (`{`) should be on the next line (Allman style, vertically aligned w/ the `if`) to match other generated code.
- Optionally, to minimize unnecessary branching in generated code:
    - don't generate the comparison when `blah` doesn't return a value
    - or do like this

    ```
    void blah_CMockStopIgnore(void)
    {
      // BEGIN generate only when blah returns a value
      if(Mock.blah_IgnoreBool)
      {
        Mock.blah_CallInstance = CMock_Guts_MemNext(Mock.blah_CallInstance);
      }
      // END   generate only when blah returns a value
      Mock.blah_IgnoreBool = (char)0;
    }
    ```

Reply:
Fixed